src/style.css

```diff
@@ -1,3 +1,69 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
+  --link_color: #00acdc;
+}
+
+.header__moyo {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+img {
+  display: block;
+}
+
+.nav__moyo {
+  height: 60px;
+}
+
+.list__moyo {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  height: 60px;
+}
```

Comment:
you should follow this instruction, also use paddings instead of hardcoding height
<img width="366" alt="Знімок екрана 2023-11-01 о 17 00 12" src="https://github.com/mate-academy/layout_moyo-header/assets/106310796/f8e4d030-db11-462c-a103-2f52c3f3386c">